packages/desktop/components/AccountManagementDetails.svelte

```diff
@@ -54,10 +54,21 @@
     $: accountId = isAccountOutput(selectedOutput) ? (selectedOutput?.output as AccountOutput)?.accountId : null
     $: address = accountId ? getBech32AddressFromAddressTypes(new AccountAddress(accountId)) : null
     $: isMainAccount = accountId && accountId === $selectedWalletMainAccountId
+    $: balance = getAccountBalance(selectedOutput)
```

Comment:
You should probably pass `isImplicitAccount` to `getAccountBalance`, so if `isImplicitAccount` changes for any reason, the balance will be updated, right?

Reply:
i'm a bit confused about why it might be necessary to pass it. If an `implicitAccount` is created through `handleActivateAccount`,  it's not like it would spontaneously change, right? 
sorry, but I don't see the situation in which this would happen... :see_no_evil:

Reply:
Yes, I agree with Eva, the moment the `implicitOutput` stops being implicit the `selectedOutput` will also change

Reply:
I prefer explicit reactive dependencies over assumptions, if L53 changes in the future, `isImplicitAccount` might start changing its value because of another factor than `selectedOutput`, and `balance` might get out of sync if you don't forget to update `getAccountBalance`. I don't see how passing `isImplicitAccount` would be a problem

Reply:
Okeey, makes sense 👌🏼